### DISCUSSION TITLE
How to support a batch of variable length sequences?

### DISCUSSION THREAD

**Discussion by deepcharles** [google/flax#3070] [General] [answered]
Hi, thanks for the great library.

I was wondering if it is possible to pass a batch of variable length sequences to a convolution layers or GRU layers? Or do we need to implement our own?

Many thanks,

Charles

**Comment by cgarciae** [comment, marked as answer]
Hey @deepcharles , to support variable length sequences is JAX you need some form of padding. You can probably use our abstractions and add padding on top.
